### PR TITLE
Research UI marker style

### PR DESCRIPTION
**Pull request by hsinlun99**
# Description
- corresponding icons for each POI status
- cluster marker icon

# Changes
- add src/assets/images/ to store icon images
- all images are svg files

# Notes
Keep "Unknown" status for debug

# Checklist

- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [X] Any changes to strings have been published to our translation tool
- [X] I conducted basic QA to assure all features are working
- [X] I requested code review from other team members

# Resolved Issues

### CODE REVIEW

File: firebase/storage.rules [hunk 1]
```diff
@@ -3,7 +3,8 @@ rules_version = '2';
 service firebase.storage {
   match /b/{bucket}/o {
     match /{allPaths=**} {
-      allow read, write: if request.auth != null;
+      allow read: if true;
+      allow write: if request.auth != null;
```
Comment (y-c-wang) [inline firebase/storage.rules:7]: Already fixed in #30.

```suggestion
      allow read, write: if request.auth != null;
```

File: src/constants/model/poi/index.ts
Comment (y-c-wang) [inline src/constants/model/poi/index.ts:1]: The result would be like this. Just to remind that you may be addressing #10 with another PR.

image

File: src/utils/googleMaps/markers/poi/icons.ts [hunk 2]
```diff
@@ -1,9 +1,53 @@
...
+import poiMarkerFunction from "../../../../assets/images/poiMarkerFunction.svg"; // 功能狀態
+import poiMarkerMaintenance from "../../../../assets/images/poiMarkerMaintenance.svg"; // 保養狀態
+import poiMarkerNoise from "../../../../assets/images/poiMarkerNoise.svg"; // 噪音狀態
+import poiMarkerOccupation from "../../../../assets/images/poiMarkerOccupation.svg"; // 占用狀態
+import poiMarkerOutlook from "../../../../assets/images/poiMarkerOutlook.svg"; // 外觀狀態
+import poiMarkerThermalComfort from "../../../../assets/images/poiMarkerThermalComfort.svg"; // 體感狀態
+import poiMarkerUsage from "../../../../assets/images/poiMarkerUsage.svg"; // 使用狀態
+import poiMarkerUnknown from "../../../../assets/images/poiMarkerUnknown.svg"; // 未知
+
+export const getIcon = (poiData: PoiData) => {
```
Comment (y-c-wang) [inline src/utils/googleMaps/markers/poi/icons.ts:14]:
```suggestion
export const getIcon = (status: PoiStatus): google.maps.Icon => {
```

File: src/utils/googleMaps/markers/poi/icons.ts [hunk 3]
```diff
@@ -1,9 +1,53 @@
...
+    case "outlook":
+      thisUrl = poiMarkerOutlook;
+      break;
+    case "thermalcomfort":
+      thisUrl = poiMarkerThermalComfort;
+      break;
+    case "usage":
+      thisUrl = poiMarkerUsage;
+      break;
+    default:
```
Comment (y-c-wang) [inline src/utils/googleMaps/markers/poi/icons.ts:44]:
```suggestion
    case "unknown":
    default:
```

File: src/constants/model/poi/index.ts [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
   unknown: "unknown",
+  clean: "clean",
+  crowd: "crowd",
+  function: "function",
+  maintenance: "maintenance",
+  noise: "noise",
+  occupation: "occupation",
+  outlook: "outlook",
+  thermalcomfort: "thermalcomfort",
+  usage: "usage",
```
Comment (y-c-wang) [inline src/constants/model/poi/index.ts:11]: Good practice to alphabetize them:)

File: src/constants/model/poi/index.ts [hunk 5]
```diff
@@ -1,17 +1,25 @@
 export const poiStatus = {
-  excellent: "excellent",
-  good: "good",
-  normal: "normal",
-  bad: "bad",
-  terrible: "terrible",
   unknown: "unknown",
```
Comment (y-c-wang) [inline src/constants/model/poi/index.ts:2]: Please reorder to put `unknown` to be the last one

**Review by y-c-wang — CHANGES_REQUESTED** [review]
Please address the comments, and remember to put your GitHub id in the front of branch name (i.e., `<your github id>/<the feature>`). Also, please attach snapshots for any UI changes.

**Comment by hsinlun99** [comment]
Thank you!
I think this PR is part of https://github.com/CAMPUS-NYCU/smart-campus/issues/10 
I will close this one and address these commits in another branch with the correct naming style

**Pull request closed without merging**